maxim/logger/llamaindex/utils.py

```
@@ -0,0 +1,83 @@
+        if message.role == MessageRole.USER:
+            return GenerationRequestMessage(role="user", content=parse_message_content(message.blocks))
+        elif message.role == MessageRole.ASSISTANT:
+            return GenerationRequestMessage(role="assistant", content=parse_message_content(message.blocks))
+        elif message.role == MessageRole.SYSTEM:
+            return GenerationRequestMessage(role="system", content=parse_message_content(message.blocks))
+        elif message.role == MessageRole.TOOL:
+            return GenerationRequestMessage(role="tool", content=parse_message_content(message.blocks))
+        else:
+            scribe().warning(f"Unsupported message role: {message.role}")
+            # TODO: Remove this once done and releasing
+            raise ValueError(f"Unsupported message role: {message.role}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)